array_connection.go

```diff
@@ -119,13 +151,35 @@ func CursorForObjectInConnection(data []interface{}, object interface{}) Connect
 	return offsetToCursor(offset)
 }
 
-func getOffset(cursor ConnectionCursor, defaultOffset int) int {
+func getOffsetWithDefault(cursor ConnectionCursor, defaultOffset int) int {
```

Comment:
Yeah, let's keep both exported, we'll try to achieve parity with `relay-js` as best as we can